code/game/objects/structures/icemoon/cave_entrance.dm

```diff
@@ -169,28 +169,26 @@ GLOBAL_LIST_INIT(ore_probability, list(
 			new /obj/item/ship_in_a_bottle(loc)
 			new /obj/item/oar(loc)
 		if(16)
-			new /obj/item/seeds/gatfruit(loc)
-		if(17)
 			new /obj/item/reagent_containers/cup/glass/drinkingglass/filled/nuka_cola(loc)
-		if(18)
+		if(17)
 			new /obj/item/soulstone/anybody(loc)
-		if(19)
+		if(18)
 			new /obj/item/disk/design_disk/modkit_disc/resonator_blast(loc)
-		if(20)
+		if(19)
 			new /obj/item/disk/design_disk/modkit_disc/rapid_repeater(loc)
-		if(21)
+		if(20)
 			new /obj/item/slimepotion/transference(loc)
-		if(22)
+		if(21)
 			new /obj/item/slime_extract/adamantine(loc)
-		if(23)
+		if(22)
 			new /obj/item/weldingtool/abductor(loc)
-		if(24)
+		if(23)
 			new /obj/structure/elite_tumor(loc)
-		if(25)
+		if(24)
 			new /mob/living/simple_animal/hostile/retaliate/clown/clownhulk(loc)
-		if(26)
+		if(25)
 			new /obj/item/clothing/shoes/winterboots/ice_boots(loc)
-		if(27)
+		if(26)
 			new /obj/item/book/granter/action/spell/sacredflame(loc)
-		if(28)
+		if(27)
```

Comment:
Oh, one thing actually

Instead of decrementing every single one of these numbers by one, can you just replace the gatfruit body with this one, and remove the branch? Makes the diffs and blames less ugly

Reply:
Better senpai?

Reply:
Yes